src/routes/analysis.py

```
@@ -3,273 +3,390 @@
+    mime = (mime_type or '').lower()
+    name = (filename or '').lower()
+
+    # Images: png/jpg/jpeg
+    if any(ext in name for ext in ('.png', '.jpg', '.jpeg')) or mime.startswith('image/'):
+        try:
+            from PIL import Image
+            import pytesseract
+            image = Image.open(BytesIO(file_bytes))
+            text = pytesseract.image_to_string(image, lang='eng+fra')
+            return text.strip()
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)